resources/zh/cemu.po

```diff
@@ -1493,337 +1849,321 @@ msgstr "更新提醒"
 
 #: MainWindow.cpp
 msgid "&Input settings"
-msgstr "输入设置"
+msgstr "&输入设置"
 
 #: MainWindow.cpp
 msgid "&Active account"
-msgstr "当前账户"
+msgstr "&当前账户"
 
 #: MainWindow.cpp
 msgid "&Console region"
-msgstr "主机地区"
+msgstr "&主机地区"
 
 #: MainWindow.cpp
 msgid "&Console language"
-msgstr "主机语言"
+msgstr "&主机语言"
 
 #: MainWindow.cpp
 msgid "&Memory searcher"
-msgstr "内存搜索"
-
-#: MainWindow.cpp
-msgid "&Tools"
-msgstr "工具"
-
-#: MainWindow.cpp
-msgid "&Single-core interpreter"
-msgstr "单核解释器"
-
-#: MainWindow.cpp
-msgid "&Single-core recompiler (fast)"
-msgstr "单核重编译器（快速）"
-
-#: MainWindow.cpp
-msgid "&Dual-core recompiler (fast, unstable!)"
-msgstr "双核重编译器（快速但不稳定）"
+msgstr "&内存搜索器"
 
 #: MainWindow.cpp
-msgid "&Triple-core recompiler (fast, unstable!)"
-msgstr "三核重编译器（快速但不稳定）"
+msgid "&Title Manager"
+msgstr "&档案管理器"
 
 #: MainWindow.cpp
-msgid "&Mode"
-msgstr "模式"
+msgid "&Tools"
+msgstr "&工具"
 
 #: MainWindow.cpp
 msgid "&CPU"
-msgstr "CPU"
+msgstr "&CPU"
 
 #: MainWindow.cpp
 msgid "&Scan NFC tag from file"
-msgstr "载入NFC数据"
+msgstr "&载入 NFC 文件"
 
 #: MainWindow.cpp
 msgid "&NFC"
-msgstr "NFC"
+msgstr "&NFC"
 
 #: MainWindow.cpp
 msgid "&Unsupported API calls"
-msgstr "不支持的API调用"
+msgstr "&不支持的 API 调用"
 
 #: MainWindow.cpp
 msgid "&Coreinit File-Access"
-msgstr "Coreinit 数据访问"
+msgstr "&Coreinit 数据访问"
 
 #: MainWindow.cpp
 msgid "&Coreinit Thread-Synchronization API"
-msgstr "Coreinit 线程同步 API"
+msgstr "&Coreinit 线程同步 API"
 
 #: MainWindow.cpp
 msgid "&Coreinit Memory API"
-msgstr "Coreinit 内存 API"
+msgstr "&Coreinit 内存 API"
+
+#: MainWindow.cpp
+msgid "&Coreinit MP API"
+msgstr "&Coreinit MP API"
+
+#: MainWindow.cpp
+msgid "&NN NFP"
+msgstr "&NN NFP"
 
 #: MainWindow.cpp
 msgid "&GX2 API"
-msgstr "GX2 API"
+msgstr "&GX2 API"
 
 #: MainWindow.cpp
 msgid "&Audio API"
-msgstr "音频 API"
+msgstr "&音频 API"
 
 #: MainWindow.cpp
 msgid "&Input API"
-msgstr "输入 API"
+msgstr "&输入 API"
 
 #: MainWindow.cpp
 msgid "&Socket API"
-msgstr "套接 API"
+msgstr "&套接 API"
 
 #: MainWindow.cpp
 msgid "&Save API"
-msgstr "储存 API"
+msgstr "&储存 API"
 
 #: MainWindow.cpp
 msgid "&H264 API"
-msgstr "H264 API"
+msgstr "&H264 API"
 
 #: MainWindow.cpp
-msgid "&NFP API"
-msgstr "NFP API"
+msgid "&Graphic pack patches"
+msgstr "&图形插件补丁"
 
 #: MainWindow.cpp
 msgid "&Texture cache warnings"
-msgstr "纹理缓存警告"
+msgstr "&纹理缓存警告"
 
 #: MainWindow.cpp
 msgid "&OpenGL debug output"
-msgstr "OpenGL 调试输出"
+msgstr "&OpenGL 调试输出"
 
 #: MainWindow.cpp
 msgid "&Vulkan validation layer (slow)"
-msgstr "Vulkan 验证层 (慢速)"
+msgstr "&Vulkan 验证层 (慢速)"
 
 #: MainWindow.cpp
 msgid "&Log PPC context for API"
-msgstr "为API记录PPC上下文"
+msgstr "&Log PPC context for API"
 
 #: MainWindow.cpp
 msgid "&Textures"
-msgstr "纹理"
+msgstr "&纹理"
 
 #: MainWindow.cpp
 msgid "&Shaders"
-msgstr "着色器"
+msgstr "&着色器"
 
 #: MainWindow.cpp
 msgid "&nlibcurl HTTP/HTTPS requests"
-msgstr "nlibcurl HTTP/HTTPS 请求"
+msgstr "&nlibcurl HTTP/HTTPS 请求"
 
 #: MainWindow.cpp
 msgid "&Logging"
-msgstr "记录"
+msgstr "&记录 Log"
```

Comment:
The following is not an English translation.
The meaning of "记录" and "log" is different in Chinese. "记录" means record, and here obviously means to record a log, so a more appropriate translation than "记录" is "记录日志".
In the Chinese context, "log" is used as a foreign noun and is often used instead of "日志", and "log" is used more frequently than "日志". So log is used here before, I will use "记录日志" to replace it later.

Reply:
Ok. The original string seemed ok to me because [Microsoft Terminology Database](https://www.microsoft.com/en-us/language/Search?&searchTerm=log&langID=124&Source=true&productid=0) translates log into 记录 (or 日志).
"Record" in English is a synonym for "log" so I find it acceptable. But feel free to use "记录日志" if it sounds more appropriate.